src/vid2805-dual-stepper-element.ts

```diff
@@ -0,0 +1,191 @@
+    inner_plain_hand:
+      'm0,0 c 0 2.6 2.1 4.7 4.7 4.7 c 1.27 0 2.41 -0.5 3.25 -1.31 h 0 c 0.5 -0.49 1.1 -0.89 1.8 -1.19 c 0.7 -0.3 1.4 -0.4 2.1 -0.4 h ${len} c 1 0 1.8 -0.8 1.8 -1.8 s -0.8 -1.8 -1.8 -1.8 h -${len} s -0.06 0 -0.06 0 c -0.7 0 -1.5 -0.1 -2.1 -0.4 c -0.7 -0.3 -1.3 -0.7 -1.8 -1.2 h 0 c -0.84 -0.8 -1.98 -1.3 -3.24 -1.3 c -2.6 0 -4.7 2.1 -4.7 4.7 Z m 2.7 0 c 0 -1.1 0.9 -2 2 -2 s 2 0.9 2 2 s -0.9 2 -2 2 s -2 -0.9 -2 -2 Z',
+    inner_arrow_hand:
+      'm 0 0 c 0 2.6 2.1 4.7 4.7 4.7 c 1.37 0 2.59 -0.59 3.44 -1.52 c 0.7 -0.7 1.5 -1.2 2.5 -1.5 s 2 -0.3 2.9 -0.1 l 0.9 0.2 l ${len} -1.8 l -${len} -1.8 l -0.9 0.2 c -1 0.2 -2 0.2 -2.9 -0.1 c -1 -0.3 -1.8 -0.8 -2.5 -1.5 c -0.86 -0.93 -2.08 -1.52 -3.44 -1.52 c -2.6 0 -4.7 2.1 -4.7 4.7 z m 2.7 0 c 0 -1.1 0.9 -2 2 -2 s 2 0.9 2 2 s -0.9 2 -2 2 s -2 -0.9 -2 -2 z',
+  };
+
+  render() {
+    let inner_svg = this.handMap['inner_' + this.innerHand.shape + '_hand'];
+    let outer_svg = this.handMap['outer_' + this.outerHand.shape + '_hand'];
+
+    // todo: validate hands
+
+    // replace length from args
+    if (this.innerHand.length > 70) {
+      this.innerHand.length = 70;
+    }
+    if (this.outerHand.length > 70) {
+      this.outerHand.length = 70;
+    }
+
+    inner_svg = inner_svg.split('${len}').join(this.innerHand.length.toString());
+    outer_svg = outer_svg.split('${len}').join(this.outerHand.length.toString());
+
+    const { x, y, innerOff, outerOff, trY } = this.coords();
+    console.debug(this.coords());
+    console.debug(trY);
+    console.debug(x);
+    console.debug(y);
+
+    return html`
+    <svg id="VID2805" xmlns="http://www.w3.org/2000/svg" width=70mm height=70mm viewBox="0 0 265 265">
+    <defs>
+        <style>
+          .cls-1{fill:#939598;}
+          .cls-2{fill:#d2d2d2;}
+          .cls-3{fill:#808285;}
+          .cls-4{fill:#ed1f24;}
+          .cls-5{fill:#70bf44;}
+          .cls-6{fill:#414042;}
+          .cls-h{fill:"blue";stroke:#000;stroke-linecap:round;stroke-linejoin:round;stroke-width:.1px;transform-box:fill-box;}
+        </style>
+        <linearGradient
+            id="hole-gradient"
+            >
+            <stop stop-color="#4f4f4f" offset="-25%" />
+            <stop stop-color="grey" offset="75%" />
+        </linearGradient>
+    </defs>
+      <g id="vid2805-translated" transform="translate(15,265) rotate(270 0 0) translate(0,${trY})">
+
+      <!-- pins - translate to top of base, between holes -->
+        <g id="pins" transform="scale(${mmToPix}) translate(21 3) ">
+          <path id="pin-1" class="cls-3" d="m 0 0 v -3 c 0 -1 1 -1 1 0 v 3 z" />
+          <use href="#pin-1" x=2.54 />
+          <use href="#pin-1" x=5.08 />
+          <use href="#pin-1" x=7.62 />
+          <use href="#pin-1" x=10.16 />
+          <use href="#pin-1" x=12.70 />
+          <use href="#pin-1" x=15.24 />
+          <use href="#pin-1" x=17.78 />
+        </g>
+
+        <g id="scaled_body" transform="scale(1.333)">
```

Comment:
ids should use kebab-case, i.e. `scaled-body`. In general, if you don't use an id, you can remove it